src/gc/gcenv.ee.standalone.inl

```diff
@@ -11,166 +11,196 @@
 // will be fowarded to this interface instance.
 extern IGCToCLR* g_theGCToCLR;
 
+// A note about this:
+// In general, we don't want to pretend to be smarter than the compiler
+// and force it to inline things. However, inlining is here is required
+// for correctness as it stands today (though it will not always be required).
```

Comment:
Not sure I follow - are you saying that it won't be a problem? When I wrote this comment I figured that `dlopen` will do the right thing with regards to conflicting symbols like this, but I'm not sure and haven't tested it.